nav2_bringup/params/nav2_params.yaml

```diff
@@ -579,3 +579,9 @@ loopback_simulator:
     scan_angle_max: 3.1415
     scan_angle_increment: 0.02617
     scan_use_inf: true
+
+pose_saver:
+  ros__parameters:
+    auto_start_saving: true
+    auto_restore_pose: true
```

Comment:
I'd disable restoring on startup, it makes it hard for simulation use where you may restart frequently and respawn back at the initial pose